src/OrchardCore.Modules/OrchardCore.Contents/Deployment/ExportContentToDeploymentTarget/ExportContentToDeploymentTargetMigrations.cs

```diff
@@ -3,12 +3,14 @@
 using OrchardCore.Data.Migration;
 using OrchardCore.Deployment;
 using OrchardCore.Entities;
+using OrchardCore.Modules;
 using OrchardCore.Recipes;
 using OrchardCore.Recipes.Services;
 using OrchardCore.Settings;
 
 namespace OrchardCore.Contents.Deployment.ExportContentToDeploymentTarget
 {
+    [Feature("OrchardCore.Contents.Deployment.ExportContentToDeploymentTarget")]
```

Comment:
This is no longer needed. It'll automatically get assigned to the feature it's being registered to.

Reply:
It doesn't look like it.

![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/7fc0486c-140f-4274-ab24-18d1ae92efd6)

Reply:
The `ExportContentToDeploymentTargetMigrations` should only be available if the "OrchardCore.Contents.Deployment.ExportContentToDeploymentTarget" feature is enabled. Are you saying without the Feature attribute the migration is available even when the feature is disabled?

Reply:
In the DataMigrationManager we request `IEnumerable<IDataMigration>` from the IoC. Since the feature is not enabled, then we should not be seeing that feature registering the data migrations.

@gvkries is it possible that this issue is due to the changes in #15793?

Reply:
I'll check it tomorrow.

Reply:
It might have something to do with my local code, I don't know where it breaks the code that recognises the migrated programs in the module

It looks normal when I debug directly using the main branch.
![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/47e141f6-0ed9-45ac-9d71-33850af44051)

Reply:
It does make a bit of a mess, but it works fine on 1.8~! 🥴